onnx/defs/operator_sets.h

```diff
@@ -1140,28 +1140,35 @@ inline void RegisterOnnxOperatorSetSchema() {
 inline void RegisterOnnxOperatorSetSchema(int target_version) {
```

Comment:
Shall we also extend an option here to further control fail_duplicate_schema in case someone is still interested in having that duplicate check?

Reply:
Done.